nofos/nofos/templates/nofos/nofo_upload_cover_image.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'base.html' %}
+{% load static nofo_name %}
+
+{% block title %}
+  Upload a cover image for "{{ nofo|nofo_name }}"
+{% endblock %}
+
+{% block body_class %}nofo_import{% endblock %}
+
+{% block content %}
+
+{% with nofo|nofo_name as nofo_name_str %}
+{% with "Back to "|add:nofo_name_str|add:"" as back_text %}
+  {% url 'nofos:nofo_edit' nofo.id as back_href %}
+  {% include "includes/page_heading.html" with title="Upload a cover image for "|add:nofo_name_str|add:"" back_text=back_text back_href=back_href only %}
+{% endwith %}
+{% endwith %}
+
+<p>
+  <strong>
+    For best results, upload an image that is at least 1000 by 1500 pixels (2:3 aspect ratio) and no larger than 5MB.
+  </strong>
+</p>
+
+<form id="nofo-upload-cover-image--form" class="form-import--loading" method="post" enctype="multipart/form-data">
+  {% csrf_token %}
+  {% with id="cover_image" label="Select an image file." hint="Accepts .jpg, .jpeg or .png files." accept=".png,.jpg,.jpeg" %}
```

Comment:
Can we remove "jpeg" from the hint (but keep it in the "accept")? We should accept it, but let's just  refer to them as "jpg".